xwiki-platform-core/xwiki-platform-rest/xwiki-platform-rest-server/src/main/java/org/xwiki/rest/internal/ModelFactory.java

```diff
@@ -956,7 +958,7 @@ public Hierarchy toRestHierarchy(EntityReference targetEntityReference, Boolean
     }
 
     /**
-     * Serializes the value of the given XObject property.
+     * Serializes the value of the given XObject property. Returns an empty string for ComputedFieldClass properties.
```

Comment:
Is the javadoc accurate? It seems that an actual value is returned for computed properties, right?

Reply:
In the current implementation of BaseProperty:getValue, the stored value of the property is returned, not the one evaluated dynamically in the case of ComputedField properties. So it may not be empty, but it's not evaluated, currently (we may consider evaluating it directly in BaseProperty:getValue but that would be another topic, I guess). I updated the Javadoc accordingly, hoping it reflects better the current situation?